src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -171,6 +171,7 @@ export default {
 							const loginHint = encodeURIComponent(
 								`login|${JSON.stringify({
 									msg: 're-auth-acc-exists',
+									guest: true,
```

Comment:
This shouldn't be used here because this is a case of an account already existing with the email address they tried to use, meaning that they aren't a guest.

The `guest: true` should be added to the login redirect in GuestAccountRedirect.vue instead

Reply:
Thanks for the review. I've moved this to the correct location in my latest commit.

Reply:
👍